tests/queries/0_stateless/01602_array_aggregation.sql

```diff
@@ -9,6 +9,9 @@ SELECT arrayMin([[3], [1], [2]]);
 SELECT 'Array max :';
 SELECT arrayMax([[3], [1], [2]]);
 
+SELECT 'Issue #69600';
+SELECT arrayMin(x1 -> x1 * -1, [1,2,3]);
+
```

Comment:
Add this test too:

select arrayMax(x -> x.1, [(1, 'a'), (0, 'b')])
result: 1